pkg/actors/actors.go

```diff
@@ -978,8 +980,13 @@ func (a *actorsRuntime) startReminder(reminder *Reminder, stopChannel chan bool)
 				break L
 			}
 			if err = a.executeReminder(reminder); err != nil {
-				log.Errorf("error execution of reminder %q for actor type %s with id %s: %v",
-					reminder.Name, reminder.ActorType, reminder.ActorID, err)
+				if err == ErrReminderCanceled {
+					// The handler is explicitly canceling the timer
+					break L
```

Comment:
Should we log this event?

Reply:
Yeah, that's probably a good idea. I'm guessing `Info` is the right level? I feel like reminders are way too chatty right now, but `Info` makes sense for consistency, I guess.